docs/source/install-macos.rst

```diff
@@ -9,6 +9,20 @@ Installation - macOS / OS X
 This guide provides instructions on installing the Cozmo SDK for computers running with a macOS operating system.
 
 ^^^^^^^^^^^^^^^^^^^
+Installation Videos
+^^^^^^^^^^^^^^^^^^^
+
+For your convenience, videos are provided showing the installation steps being followed on a macOS / OS X computer. One using an iOS device, and one using an Android device. There is also full text-based documenation below these.
```

Comment:
Correct spelling: "documentation"

The sentence "One using an iOS device, and one using an Android device", should be part of the previous sentence, and shouldn't be a sentence on its own.